docker-compose.yml

```diff
@@ -21,9 +21,9 @@ services:
       - DB_HOST=db
       - ALLOWED_HOSTS=["*"]
       - CORS_ORIGIN_WHITELIST=('localhost:3000',)
-      - GATHER_CASES_CRON_TIME_SECONDS=1440
-      - GATHER_VOTES_CRON_TIME_SECONDS=1440
-      - LINK_CASES_AND_PROMISES_CRON_TIME_SECONDS=1440
+      - GATHER_CASES_CRON_TIME_SECONDS=1
```

Comment:
did you revert this in a future commit?
Maybe keep these changes out of the commit history in the future?

Reply:
ya reverted